index.js

```diff
@@ -23,7 +23,7 @@ const addEventToElement = (element, eventKey, eventValue) => {
 // if something gets processed, we return false
 // otherwise, it returns true, indicating that this thing needs to be processed
 const handleEventSetter = element => prop => prop.key.slice(0, 2) === 'on' ? addEventToElement(element, prop.key, prop.value) : true
-const handleAttrSetter = element => prop => element.setAttributeNS(null, prop.key, prop.value)
+const handleAttrSetter = element => prop => prop.key.match(/:/) ? element.setAttribute(prop.key, prop.value) : element.setAttributeNS(null, prop.key, prop.value)
```

Comment:
If this is a custom namespace attribute (like `context:list`) we use the non-namespace aware method `element.setAttribute`